elytron/src/test/java/org/wildfly/extension/elytron/TlsTestCase.java

```diff
@@ -471,6 +494,21 @@ public void testRevocationListsDp() throws Throwable {
         Assert.assertTrue(services.executeOperation(operation).get(OUTCOME).asString().equals(FAILED)); // not realoadable
     }
 
+    /**
+     * Tests distribution points work using certificate-revocation-lists attribute
+     */
+    @Test
+    public void testCertificateRevocationListsDp() throws Throwable {
+        ServiceName serviceName = Capabilities.TRUST_MANAGER_RUNTIME_CAPABILITY.getCapabilityServiceName("trust-with-crls-dp");
+        TrustManager trustManager = (TrustManager) services.getContainer().getService(serviceName).getValue();
+        Assert.assertNotNull(trustManager);
+
+        ModelNode operation = new ModelNode();
+        operation.get(ClientConstants.OP_ADDR).add("subsystem", "elytron").add(ElytronDescriptionConstants.TRUST_MANAGER, "trust-with-crls-dp");
+        operation.get(ClientConstants.OP).set(ElytronDescriptionConstants.RELOAD_CERTIFICATE_REVOCATION_LIST);
+        Assert.assertTrue(services.executeOperation(operation).get(OUTCOME).asString().equals(FAILED)); // not realoadable
+    }
+
```

Comment:
I think it would be good to add some tests cases here to assert that the configured CRLs actually get used successfully.

Reply:
I added a one way test to ensure the CRLs are being used correctly.